src/ocrd/decorators/__init__.py

```diff
@@ -107,10 +107,10 @@ def resolve(name):
     if 'parameter_override' in kwargs:
         set_json_key_value_overrides(kwargs['parameter'], *kwargs.pop('parameter_override'))
     # Assert -I / -O
-    if not kwargs['input_file_grp']:
+    if 'input_file_grp' not in kwargs:
```

Comment:
```suggestion
    if not kwargs.get('input_file_grp', None):
```

as discussed

Reply:
`ocrd-olahd-client` requires neither input nor output. But since that is a fairly esoteric processor that no one seems to be using, probably not reason enough to change that behavior in spec/core.

@MehmedGIT @j-panzer Is the OLA-HD client still in use?

Reply:
@kba, no it is not used by Operandi nor the Integration script.

Reply:
Perhaps we could treat this as a special case: do require an input fileGrp, but only for informational purposes or to represent workflow dependency (final fileGrp), not for actually doing something specific with that fileGrp.

So instead of `input_file_grp_cardinality: 0` this would become 1.

It would require users specify an input fileGrp, but that should normally not be a problem – either pass the "maximal" fileGrp of the workflow, or just use `OCR-D-IMG`.

(Just to make sure we don't bend the spec because of this.)